storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -6498,6 +6698,78 @@ static void xtrabackup_prepare_func(int argc, char **argv) {
 
   Tablespace_map::instance().deserialize("./");
 
+  if (opt_lock_ddl == LOCK_DDL_REDUCED) {
+    if (!xb_process_datadir(
+            xtrabackup_incremental_dir ? xtrabackup_incremental_dir : ".",
+            ".corrupt", prepare_handle_corrupt_files, NULL)) {
```

Comment:
new extention for ".CORRUPT". May be introduced a short name ".CRPT" ?

Reply:
AB